indent-lint.el

```diff
@@ -5,7 +5,7 @@
 ;; Author: Naoya Yamashita <conao3@gmail.com>
 ;; Version: 1.1.4
 ;; Keywords: tools
-;; Package-Requires: ((emacs "25.1") (async-await "1.0") (async "1.9.4"))
```

Comment:
I think the missing closing paren in this line is the reason why this package is not building in MELPA, and that flycheck-indent is consequently marked as broken.

Reply:
Thanks for reporting this! this issue is resolved at https://github.com/conao3/indent-lint.el/pull/38